2007-11-19  Aurelien Jarno  <aurelien@aurel32.net>

	* malloc/malloc.h: Define __need_NULL, __need_ptrdiff_t
	and__need_size_t.

Index: malloc/malloc.h
===================================================================
RCS file: /cvs/glibc/libc/malloc/malloc.h,v
retrieving revision 1.30
diff -u -d -p -r1.30 malloc.h
--- malloc/malloc.h	19 Jul 2007 17:05:07 -0000	1.30
+++ malloc/malloc.h	28 Nov 2007 07:57:02 -0000
@@ -22,6 +22,9 @@
 #define _MALLOC_H 1
 
 #include <features.h>
+#define __need_NULL
+#define __need_ptrdiff_t
+#define __need_size_t
 #include <stddef.h>
 # define __malloc_ptr_t  void *
 
